### DISCUSSION TITLE
Lots of people losing confidence in Chia now. Here is the thing

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#6711] [General]
I do suggest Chia officially post a rumor refute article in multiple languages (especially Chinese).
So many rumors surrounding me and the entire Chia community. We need some confidence.
Here is the list of what I think should be include. Some of them are my explain and the others are the questions from the community.

1. Fake plots
> There is a way that generates some metadata of plots and cheating. It won't affect the network since it is unable to submit any proof. However, when people seeing there is a fake capacity that has been recognized, they will trust it. Not many people have a background in technical stuff. The fake plots may able to cheat on HPool or similar pools which are **not using official pooling protocol**. Chia officials do not recommend using the pools like that and will never respond to the non-official pooling protocol. People who use these unsafe pools need to take care of themselves. Fake plots and copied plots will not count to the total net space and the total net space is real.
2. New plotter
> The MADMAX plotter does have some improvement, but it is not a huge difference from the original plotter. The MADMAX plotter will utilize all the CPU cores with a single plotting task. The original plotter will not use all the resources with a single plotting task. So it will be able to parallel more plotting tasks while the MADMAX plotter can't. However, the MADMAX plotter does have many improvements and can improve the overall speed by 10-40% (in my observation). It won't harm Chia network but benefit Chia network.
3. Marketing
> People wondering how would the Chia team promote Chia. Not seeing any advertisements/articles yet.
4. When pool
> Official pooling protocol is developing and all the pool operators are working on their pools. Chia official will not tell any TBD and it will just be there if it finished.
5. What did you do in the past 3 years
> People are annoying that why Chia looks so unmatured right now. So many bugs in the client and no pooling support. It should not be the work in 3 years.
6. Python?
> People do not trust Chia software because it is written in Python. Chia officials might need to clarify that Chia core is written by CPP which is in the `chiapos` repo. Python/HTML/JS/TS/etc. are the tools to make life easier.
7. 30sec threshold
> Farming Chia should finish a challenge under 30sec window. This 30sec including the time on network, calculation, disk seeking and reading, etc. That is why proof seeking would be better under 5sec. If the total time under 30sec, it will be fine. Over 30sec will lose the prize.
8. Fake official
> There is no Chia official group/chat/team/channel/forum other than Chia website, Github, and Keybase. No underground trade or collaboration with any person/organization. No official anything in China.
9. Chia network is safe
> Letting people know Chia network is still safe and there is no attack or cheating on the network.
10. XCH price
> It continues going down down down. Any idea?

Wish these questions could be answered by Chia official and publish. It will make a lot of people have a good sleep.
I volunteer to be a translator if Chia official needs.

**Comment by loppefaaret** [comment]
we do have an AMA planned for tuesday, on some of the recent concerns

**Reply by Philipmcnill** [reply]
I do not have "pool" icon on my GUI. What I need to install to get it?

**Reply by cccat6** [reply]
Not released yet. You need some patience.

**Reply by Philipmcnill** [reply]
Yeah, but I saw on YT and people already have it. Is that some testing versions?

**Reply by cccat6** [reply]
That's not related to anyone other than developers or pool operators.
Nobody can use that on the mainnet until it releases.
Nobody can get any benefit from testnet version.

**Comment by iaimit** [comment]
The official delay in releasing the pool agreement!

**Comment by alexwwstx** [comment]
The cost to mine and keep plots is pretty high and could trigger a lot long term confidence concerning provided the current HDD price, XCH value, and difficulty to get XCH mined out by small farmer.
A good / bad sign is HDD price starts to drop...

**Comment by ghost** [comment]
For me its looks fake and nothing is official data yet and no help for small farmer everyone with any number of plot whether its 1 or 100000 everyone should get something else its just old shit of piece software to scam peoples computer power to gain by company for mining, i have 1034 plots since more then month and i have 0 chia win to date, whats the use to plot.

**Reply by alexwwstx** [reply]
better to join a pool to see tiny coming every day rather than a big cake one day. :-)

**Reply by Philipmcnill** [reply]
### Check your harvester
Check your log file for lines like that:  **2021-06-24T08:09:58.084 harvester chia.harvester.harvester: INFO     2 plots were eligible for farming 66a6c0711e... Found 0 proofs. Time: 0.17187 s. Total 1277 plot**s. and follow this video https://www.youtube.com/watch?v=wOLakMJDLTw

However, I also personally think, this projects is not well designed, not clear and totally out of original conception. The "Green Crypto" idea in this case hasn't included waste of massive SSD usage. To keep my farm on cost me around 14$ per month and energy is not green as my energy is generated by coal power plant. Over 30% Chia "factories" are in China and they get most of rewards. The only money were made by big Chia "factories", YouTubers and HD manufacturers.

**Reply by ghost** [reply]
> 
> 
> better to join a pool to see tiny coming every day rather than a big cake one day. :-)

there is no offical pool yet and the others are you cant trust they can kick u out and eat all cake alone

**Reply by ghost** [reply]
> 
> ### Check your harvester
> 
> Check your log file for lines like that: **2021-06-24T08:09:58.084 harvester chia.harvester.harvester: INFO 2 plots were eligible for farming 66a6c0711e... Found 0 proofs. Time: 0.17187 s. Total 1277 plot**s. and follow this video https://www.youtube.com/watch?v=wOLakMJDLTw
> 
> However, I also personally think, this projects is not well designed, not clear and totally out of original conception. The "Green Crypto" idea in this case hasn't included waste of massive SSD usage. To keep my farm on cost me around 14$ per month and energy is not green as my energy is generated by coal power plant. Over 30% Chia "factories" are in China and they get most of rewards. The only money were made by big Chia "factories", YouTubers and HD manufacturers.

there is no issue with log and in addition to it im using https://github.com/martomi/chiadog  for monitoring, all seems good.

**Reply by Philipmcnill** [reply]
Thanks for link I will also check it

**Reply by Philipmcnill** [reply]
I feel pain when I think to make all this plots again when the lunch pools. Any idea when it may happen?

**Reply by ghost** [reply]
the official staff/admins/mods/ are ghost. they launched the project and resting around seeing netsapce grows until when they will get there data and analyze date to invest ton another technologies and abandoned this project.

**Reply by loppefaaret** [reply]
i don't why you ended up with that point of view- that is about as far away from the truth as we want to be perceived...

**Reply by ghost** [reply]
why there is no foolproof way to win block when you guys promised, you left all small farmers left behind only people with data centers are doing farming now, even people with 10,000+ plots considered small farmers now.

Why should we farm when there is no rerun and there is no solution coming.

**Reply by loppefaaret** [reply]
there is a solution hitting the coming main testnet - we are entering the test phase of official pooling protocol
i understand your frustration, but to say we are doing nothing, and just leaning back, and waiting to cash out is so far from reality as it can be, we got every single capable software engineer on the pooling project, and they are putting serious amounts of hours into it

**Reply by ghost** [reply]
Thank you for answer it. as now all farming done by big investors and we still at loss with fake promises to win block with estimate time to win  even the estimate time is 3-4 times more then showing and there is no guaranteed for winning.  at the end its not Green Digital Mining, my pc run 24x7 which cost bills to operate in return we have 0 ROI. Please think

**Reply by Philipmcnill** [reply]
k-for is right! People who believed in project feels left with no attention. There is no even updates as it used to be on the main Chia website.  I run two machines 24x7 it costs me around 26 Euros per month, I'm wondering what is "Green" hear :)

**Reply by ghost** [reply]
Chia Logo which i think they called Green.

**Reply by loppefaaret** [reply]
farming is done by everyone on the blockchain, the current netspace is 26½EiB - all of that is farming, but only one proof can win a reward every 18.75 seconds... every round is equal opportunity for every proof on the network... there might be a 6PiB farmer that has won 3 times more than he should the last day... and there might be a 80 TiB guy that has yet to win... but that is the nature of crypto... 
no one expected this to be even a 1/10 of what it is today, when it comes to the amount of netspace that is active after 3 months
on the question of making it more likely to win when you are a small farmer, we have chosen to reward strictly equal to the amount of space you fill with plots - and if we start changing that now, im pretty sure all the big farmers will just split their farms into smaller pieces instead, appearing as small farms, optimizing their revenue

**Reply by loppefaaret** [reply]
on the subject of regular updates, we are working on a better way to keep the community informed of ongoing processes, and updates on a regular interval - and you are absolutely right, that is an area we can, and want to be, better at!

**Reply by ghost** [reply]
> 
> 
> farming is done by everyone on the blockchain, the current netspace is 26½EiB - all of that is farming, but only one proof can win a reward every 18.75 seconds... every round is equal opportunity for every proof on the network... there might be a 6PiB farmer that has won 3 times more than he should the last day... and there might be a 80 TiB guy that has yet to win... but that is the nature of crypto...
> no one expected this to be even a 1/10 of what it is today, when it comes to the amount of netspace that is active after 3 months
> on the question of making it more likely to win when you are a small farmer, we have chosen to reward strictly equal to the amount of space you fill with plots - and if we start changing that now, im pretty sure all the big farmers will just split their farms into smaller pieces instead, appearing as small farms, optimizing their revenue

So you want to say is that i should give up with my 1000+ plots which never won anything or if i want to win then invest more and add more plots without any return, its not what supposed we get in return.

**Reply by loppefaaret** [reply]
what you choose to do, is up to you, but i would personally not give up now, we are just getting started, and plan to be around for many years to come!

**Reply by room101-dev** [reply]
> > better to join a pool to see tiny coming every day rather than a big cake one day. :-)
> 
> there is no offical pool yet and the others are you cant trust they can kick u out and eat all cake alone

That's really funny, I have been on HPOOL for 3 months now getting more XCH everyday. It's simply pathetic that the CHIA users are self-inflected losers; The kind of people that should have stayed home forever in the crib, 

**Even the new "FLAX" clone of chia is bug free, and has working pools sponsored by foxypool.io,** so there is now no reason to 'wait for mother chia', she's a woman of ill repute, and she's never going to come home to take care of you.

Most of us called out 'Chia-Dev' some 3+ months ago as being incompetent, and would never deliver; Well we were right.

**Reply by ghost** [reply]
HPOOL is old story which closed registration a while ago. you was adopted by hpool to take care from orphan shelter so you was lucky.
